### ISSUE TITLE
I changed model input logits shape as  (64，128，128，1) ， labels shape as (64，128，128，1)  for binary classify ( I do not want to use one hot label )，which loss function will work？  I used tf.nn.sigmoid_crossentropy_with_logits() but did not work，refer to tensorflow document   which explain logits has same shape to labels ，I am wrong?  and your lovazs loss not work for refinenet  resnet50  binary classify( I didnt try other model)

### ISSUE THREAD

**Issue by liuchuanloong** [GeorgeSeif/Semantic-Segmentation-Suite#98]
Please fill out this issue template before submitting. **Issues which do not fill out this template, or are already answered in the FAQs will simply be closed.**

**Please go to Stack Overflow for help and support.** Also check past issues as many are repeats. Also check out the Frequently Asked Questions (FAQs) below in case your question has already been answered in an issue!

Issues should be one of the following:

1. Feature requests
2. Bug reports

------------------------

## Please specify the following information when submitting an issue:
- **What are your command line arguments?:**
- **Have you written any custom code?**:
- **What have you done to try and solve this issue?:**
- **TensorFlow version?**:

### Describe the problem
Describe the problem clearly here. Be sure to convey here why it's a bug or a feature request.

### Source code / logs
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Try to provide a reproducible test case that is the bare minimum necessary to generate the problem.

### FAQs

- **Question:** I got an `InvalidArgumentError` saying that `Dimensions of inputs should match` **Answer:** See issue #17

- **Question:** Can you upload pre-trained weights for these networks? **Answer:** See issue #57

- **Question:** Do I need a GPU to train these models? **Answer:** Technically no, but I'd highly recommend it. I was able to train the models pretty well in about a day using a 1080Ti GPU. Training on CPU would take much longer than that.

- **Question:** Will you be adding the FCN or U-Net models? **Answer:** No I won't be adding those simply because they're a few years old and state-of-the-art has moved past that.

- **Question:** I got an invalid argument error when using the InceptionV4 model. Am I doing something wrong? **Answer:** No you're not! Due to the design of the InceptiveV4 model, when you end up upsampling you do some rounding which creates a shape mismatch. _This only happens when you end up having to use the `end_points['pool5']`_. See the code for some of the models if you want to check whether the model will use `end_points['pool5']`.

**Comment by GeorgeSeif** [comment]
Hi there, could you please fill out the issue template?

Thanks!

**Comment by GeorgeSeif** [comment]
OK, personally, I've never tried to do it without one-hot labels since one-hot is the standard way for any classification task, though you could probably do it as a regression as you intend to. You'd just have to round the end result!

But why not do one hot labels? It'll work just fine for binary cases too :)

**Comment by liuchuanloong** [comment]
I have to train my lovasz loss without one hot label, and i would like load pretrain model in binary loss ,i hope last layer parameters will be load

**Comment by GeorgeSeif** [comment]
Ah I see. Well you can always load the layers by name, that way, it'll load all the layers that have the right corresponding names and ignore the ones that don't! It should train just fine that way :)

**Comment by liuchuanloong** [comment]
I dont known implement by tensorflow , in keras  it can out-of-box, let me try

**Comment by GeorgeSeif** [comment]
Closing due to lack of recent activity
